engines/griffon/cutscenes.cpp

```diff
@@ -367,15 +407,25 @@ void GriffonEngine::endOfGame() {
 		_titleImg->blit(*_videoBuffer, rc.left, rc.top);
 
 		y = y - spd * _fpsr;
-		for (int i = 0; i <= 26; i++) {
+
+		for (int i = 0; i < ARRAYSIZE(story2); i++) {
 			int yy = y + i * 10;
+
+#ifdef USE_TTS
+			if (i == nextparagraph)
+				nextparagraph = textToSpeech(nextparagraph, story2, ARRAYSIZE(story2));
+#endif
+
 			if (yy > -8 && yy < 240) {
 				int x = 160 - strlen(story2[i]) * 4;
 				drawString(_videoBuffer, story2[i], x, yy, 4);
 			}
 
-			if (yy < 10 && i == 25)
-				break;
+			if (yy < 10 && i == ARRAYSIZE(story2)-1) {
+				if (ttsMan != nullptr)
+					if (ttsMan->isSpeaking() == false)
+						break;
+			}
```

Comment:
Same comment as above. I would expect to break as well is `ttsMan` is a nullptr.

Reply:
Why change the `break` into a `return` here? Is this a mistake?